src/main/java/com/scalar/database/transaction/consensuscommit/CommitHandler.java

```diff
@@ -18,21 +18,41 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/** Handler for the commit phase of a transaction. */
```

Comment:
I think here is the place to explain how a transaction work in scalar db. Please carefully read the paper, understand it and come back.